docs/index.bs

```diff
@@ -1725,11 +1557,6 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
       </thead>
       <tbody>
         <tr>
-          <td><dfn event id="service-worker-global-scope-install-event"><code>install</code></dfn></td>
```

Comment:
This block should still be there, right? Just with ExtendableEvent rather than InstallEvent

Reply:
Ah yes, correct